### PR TITLE
UAF-0087 / UAF-3237 MOD-CG007 Attributes on Agency

### PR DESCRIPTION
**Pull request by adamkost**

### CODE REVIEW

File: kfs-cg/src/main/resources/edu/arizona/kfs/module/cg/businessobject/datadictionary/Agency.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:p="http://www.springframework.org/schema/p"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+    <bean id="Agency-fullName" parent="Agency-fullName-parentBean">
+        <property name="maxLength" value="200" />
```
Comment (ghost) [inline kfs-cg/src/main/resources/edu/arizona/kfs/module/cg/businessobject/datadictionary/Agency.xml:8]: Please add:
`<property name="control">
	<bean parent="TextControlDefinition" p:size="200" />
</property>`

File: kfs-core/src/main/resources/edu/arizona/kfs/integration/cg/businessobject/datadictionary/Agency.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:p="http://www.springframework.org/schema/p"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+    <bean id="Agency-fullName" parent="Agency-fullName-parentBean">
+        <property name="maxLength" value="200" />
```
Comment (ghost) [inline kfs-core/src/main/resources/edu/arizona/kfs/integration/cg/businessobject/datadictionary/Agency.xml:8]: Please add:
`<property name="control">
	<bean parent="TextControlDefinition" p:size="200" />
</property>`

File: kfs-kc/src/main/resources/edu/arizona/kfs/module/external/kc/businessobject/datadictionary/Agency.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:p="http://www.springframework.org/schema/p"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+    <bean id="Agency-fullName" parent="Agency-fullName-parentBean">
+        <property name="maxLength" value="200" />
```
Comment (ghost) [inline kfs-kc/src/main/resources/edu/arizona/kfs/module/external/kc/businessobject/datadictionary/Agency.xml:8]: Please add:
`<property name="control">
	<bean parent="TextControlDefinition" p:size="200" />
</property>`

**Review by ghost — COMMENTED** [review]
Initial code review complete. Please see my comments.

File: kfs-cg/src/main/resources/edu/arizona/kfs/module/cg/businessobject/datadictionary/Agency.xml (hunk 1 above)
Comment (adamkost) [reply kfs-cg/src/main/resources/edu/arizona/kfs/module/cg/businessobject/datadictionary/Agency.xml:8]: done.

File: kfs-core/src/main/resources/edu/arizona/kfs/integration/cg/businessobject/datadictionary/Agency.xml (hunk 2 above)
Comment (adamkost) [reply kfs-core/src/main/resources/edu/arizona/kfs/integration/cg/businessobject/datadictionary/Agency.xml:8]: done.

File: kfs-kc/src/main/resources/edu/arizona/kfs/module/external/kc/businessobject/datadictionary/Agency.xml (hunk 3 above)
Comment (adamkost) [reply kfs-kc/src/main/resources/edu/arizona/kfs/module/external/kc/businessobject/datadictionary/Agency.xml:8]: done.

**Comment by ghost** [comment]
Looks good!

**Pull request merged**